cpp/ps0000/ps0000_slon.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream> // 1/2
```

Comment:
- [ ] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`